harness/determined/common/experimental/determined.py

```diff
@@ -155,25 +155,23 @@ def create_model(
 
         return model.Model._from_json(r.json().get("model"), self._session)
 
-    def get_model(self, name: str) -> model.Model:
+    def get_model(self, identifier: Union[str, int]) -> model.Model:
         """
         Get the :class:`~determined.experimental.Model` from the model registry
-        with the provided name. If no model with that id is found in the registry,
+        with the provided name or ID. If no corresponding model is found in the registry,
```

Comment:
perhaps clarify to say `with the provided identifer, which is either a string-type name or an integer-type model ID`.

Reply:
✅